src/types/index.tsx

```diff
@@ -0,0 +1 @@
+export type Language = "de" | "fr" | "it" | "en"
```

Comment:
very generic filename that will potentially attract a multiverse of different types. what about going for something more specific like `types/i18n.ts`?

(I realize just now that it's also a tsx file that is no component)

Reply:
That was the idea to have a place to put shared types, no pref will rename and change the file